mui/utils.py

```diff
@@ -112,3 +115,64 @@ def print_timestamp(*args, **kw):
     """Print with timestamp prefixed (local timezone)"""
     timestamp = datetime.now().astimezone()
     print(f"[{timestamp}]", *args, **kw)
+
+
+@dataclass
+class MUIFunctionModel:
+    name: str
+    func: typing.Callable
+
+
+def get_function_models() -> typing.List[MUIFunctionModel]:
+    """
+    Returns available function models
+    ref: https://github.com/trailofbits/manticore/blob/master/docs/native.rst#function-models
+    """
+
+    # Functions only
+    functions = filter(lambda x: isfunction(x[1]), getmembers(models))
+    func_models = [MUIFunctionModel(name, func) for name, func in functions]
+
+    # Manually remove non-function model functions
+    def is_model(model: MUIFunctionModel) -> bool:
+        blacklist = set(["isvariadic", "variadic", "must_be_NULL", "cannot_be_NULL", "can_be_NULL"])
```

Comment:
Oh, where do these identifiers come from? Binary Ninja?

Reply:
Right now I'm getting all the function models from manticore by
1. Listing all functions from models.py
2. Removing non model functions (helpers etc.) manually

So this blacklist is manually created from the functions in models.py that are not models.

Reply:
https://github.com/trailofbits/manticore/blob/master/manticore/native/models.py

Reply:
One of the reasons I used this method instead of manually creating a list of functions models is so that we wouldn't have to update MUI code to update for each new function model being added (assumption being that blacklisted functions would change less often than "whitelisted" (function models) functions.

If we are going to use this approach to test should I change the implementation to just use a manual list of function models instead?